components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2UserOperationEventPayloadBuilder.java

```diff
@@ -140,6 +140,11 @@ public EventPayload buildUserDeleteEvent(EventData eventData) throws IdentityEve
     @Override
     public EventPayload buildUserUnlockAccountEvent(EventData eventData) throws IdentityEventException {
 
+        return this.buildUserAccountEvent(eventData);
+    }
+
+    private EventPayload buildUserAccountEvent(EventData eventData) throws IdentityEventException {
```

Comment:
Looks like you have utilized this method from two to build payload for account lock and account unlock.
But within the method variable names used doesn't indicate that and make it more confusing. For example, the user is referenced as 'unlockedUser'